bin/ch-run-oci

```diff
@@ -79,27 +80,30 @@ def args_parse():
       cmd = CH_BIN + "/ch-run"
       os.execl(cmd, cmd, "--version")
 
-   bundle_ = bundle_from_cid(args_.cid)
+   bundle_ = bundle_from_cid(args_.cid, BUNDLE_PREFIX)
```

Comment:
Just use the `BUNDLE_PREFIX` constant directly in `bundle_from_cid()`, without adding an argument (unless I'm missing something).